common/src/main/aidl/com/microsoft/identity/client/IMicrosoftAuthService.aidl

```diff
@@ -38,6 +38,8 @@ interface IMicrosoftAuthService {
 
     Intent getIntentForInteractiveRequest();
 
+    Intent getIntentForATv2InteractiveRequest();
```

Comment:
>ATv2 [](http://example.com/codeflow?start=23&length=4)

AccountTransferV2 instead of ATv2?

Reply:
Sure thing

Reply:
renamed all references to ATv2 in both PRs